docs/source/tutorial/schema.md

```diff
@@ -204,11 +204,41 @@ server.listen().then(({ url }) => {
 
 After saving, run `npm start` to start your server! 🎉 Apollo Server is now available on port 4000.
 
-### Explore your schema
+## Explore your schema
 
-With your server running, visit `localhost:4000` in your browser to open [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/features/graphql-playground/), an IDE that enables you to **introspect** your schema and test out queries. Introspection is a helpful feature of a GraphQL server that enables you to obtain its schema.
+Now that our server is running, we can **introspect** its schema types and fields with useful developer tools. Later, we'll also be able to run test queries from those tools.
 
-> Note that introspection should be **disabled** for a production GraphQL server. Apollo Server disables introspection automatically if the `NODE_ENV` environment variable is set to `production`.
+> Introspection is a helpful GraphQL feature that enables you to obtain a server's schema for development purposes. It should be **disabled** for servers running in production. Apollo Server disables introspection automatically if the `NODE_ENV` environment variable is set to `production`.
+
+### With Apollo Studio (recommended)
+
+The **Apollo Studio Explorer** is a powerful free web IDE for exploring your GraphQL schema and building queries against it:
+
+<img class="screenshot" src="../img/explorer-tab.jpg" />
+
+To use the Explorer with your locally running server, you register a **dev graph** with Apollo Studio, like so:
```

Comment:
Do we want to use the word "register" here? Maybe saying "create" will feel a bit lighter? We've generally been avoiding trying to introduce the registry concept with dev graphs to keep things as simple as possible.